tests/testMongo.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+const BaseController = require('../api/base-controller');
+const MongoDbModel = require('../api/mongodb-model');
+const HappyCrud = require('../api');
+const BaseTest = require('./base-test');
+const should = require('should');
+const mongoose = require('mongoose');
+const events = require('events');
+const util = require('util');
+
+class ControllerMongo extends BaseController {
+} // class ModelB
+
+class TestMongo extends BaseTest {
+
+constructor(server) {
+  super(server);
+  var self = this;
+
+  const schema = {
+    a: String,
+    b: String,
+    c: Number,
+  }
+
+  const options = {
+    word: 'mongo',
+    path: '/api',
+    mount: '/',
+  }
+
+  const col = 'testb';
+  const model = new MongoDbModel(col, schema);
+  const ctrl = new ControllerMongo(model);
+  const api = new HappyCrud(server, ctrl, options);
+
+  // Setup db and return instance in promise
+  mongoose.connect('mongodb://localhost/test');
```

Comment:
Can these lines be moved to the model? So we can install the events on the model and the users wouldn't need to bother setting up event emitters by themselves.